ci/setup-build-test-gdk.sh

```diff
@@ -13,10 +13,10 @@ pushd "$(dirname "$0")"
     BUILD_HOME="${3:-"$(pwd)/../.."}"
 
     UNREAL_PATH="${BUILD_HOME}/UnrealEngine"
-    TEST_UPROJECT_NAME="EngineNetTest"
-    TEST_REPO_URL="git@github.com:improbable/UnrealGDKEngineNetTest.git"
-    TEST_REPO_MAP="NetworkingMap"
-    TEST_PROJECT_NAME="NetworkTestProject"
+    TEST_UPROJECT_NAME="UnrealGDKTestGyms"
```

Comment:
I think this will conflict with Jess's changes now.

Reply:
Hopefully easily resolved once they have to be integrated :)